### ISSUE TITLE
Question: Only modify the audio source . NOT the video source. 

### ISSUE THREAD

**Issue by DJuego** [staxrip/staxrip#222]
Context:

 My Samsung Smart TV C6530 doesn't play a very few video chapter .mp4 files from a serie. My TV tells me that the problem is in the audio codec.  I have been comparing working and non-working chapters using the MediaInfo program.  And so...

Do not work: 89,0 Kb/s 48.0 Khz 2 channels AAC LC SBR (NBC)(HE-AAC)
Works: 89,0 Kb/s 48.0 Khz 2 channels AAC LC SBR (Explicit)(HE-AAC)

How do I convert to playable .mp4 files? Any suggestion? Is it possible NO re-encode video with staxrip (just use the direct video stream)?. I only want to re-encode audio. I wish all the chapters in the series had the same configuration. And the video plays perfectly. The file size is already perfect and I don't want to risk quality losses

Thanks!

More comprehensive description:

**it doesn't work.**

Format                                   : MPEG-4
Format profile                           : Base Media / Version 2
Codec ID                                 : mp42 (isom/iso2/avc1/mp41)
File size                                : 121 MiB
Duration                                 : 23 min 39 s
Overall bit rate mode                    : Variable
Overall bit rate                         : 717 kb/s
Encoded date                             : UTC 2036-02-06 06:28:16
Tagged date                              : UTC 2036-02-06 06:28:16
Writing application                      : HandBrake 0.10.2 2015060900

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4
Format settings                          : CABAC / 9 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 9 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 23 min 39 s
Bit rate                                 : 633 kb/s
Width                                    : 1 280 pixels
Height                                   : 720 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 23.976 (24000/1001) FPS
Minimum frame rate                       : 23.974 FPS
Maximum frame rate                       : 23.981 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.029
Stream size                              : 107 MiB (88%)
Writing library                          : x264 core 142 r2479 dd79a61
Encoding settings                        : cabac=1 / ref=9 / deblock=1:1:1 / analyse=0x3:0x113 / me=umh / subme=9 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=1 / me_range=24 / chroma_me=1 / trellis=2 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=48 / lookahead_threads=5 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=9 / b_pyramid=2 / b_adapt=2 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=2 / keyint=240 / keyint_min=24 / scenecut=40 / intra_refresh=0 / rc_lookahead=60 / rc=2pass / mbtree=1 / bitrate=633 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / cplxblur=20.0 / qblur=0.5 / vbv_maxrate=25000 / vbv_bufsize=31250 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:1.00
Encoded date                             : UTC 2036-02-06 06:28:16
Tagged date                              : UTC 2036-02-06 06:28:16
Color range                              : Limited
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709
Codec configuration box                  : avcC

Audio
ID                                       : 2
Format                                   : AAC LC SBR
Format/Info                              : Advanced Audio Codec Low Complexity with Spectral Band Replication
Commercial name                          : HE-AAC
Format settings                          : NBC
Codec ID                                 : mp4a-40-5
Duration                                 : 23 min 39 s
Bit rate mode                            : Variable
Bit rate                                 : 80.0 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 23.438 FPS (2048 SPF)
Compression mode                         : Lossy
Stream size                              : 13.5 MiB (11%)
Title                                    : Stereo / Stereo
Default                                  : Yes
Alternate group                          : 1
Encoded date                             : UTC 2036-02-06 06:28:16
Tagged date                              : UTC 2036-02-06 06:28:16

**it works!**

Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/avc1)
File size                                : 120 MiB
Duration                                 : 23 min 40 s
Overall bit rate mode                    : Variable
Overall bit rate                         : 709 kb/s
Encoded date                             : UTC 2018-04-08 12:00:12
Tagged date                              : UTC 2018-04-08 12:00:12

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4
Format settings                          : CABAC / 6 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 6 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 23 min 39 s
Bit rate                                 : 626 kb/s
Maximum bit rate                         : 3 911 kb/s
Width                                    : 1 280 pixels
Height                                   : 720 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.028
Stream size                              : 106 MiB (88%)
Writing library                          : x264 core 133 r2334 a3ac64b
Encoding settings                        : cabac=1 / ref=6 / deblock=1:1:1 / analyse=0x3:0x133 / me=umh / subme=9 / psy=1 / psy_rd=0.40:0.00 / mixed_ref=1 / me_range=24 / chroma_me=1 / trellis=2 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=48 / lookahead_threads=5 / sliced_threads=0 / nr=300 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=6 / b_pyramid=2 / b_adapt=2 / b_bias=0 / direct=3 / weightb=1 / open_gop=0 / weightp=2 / keyint=240 / keyint_min=23 / scenecut=40 / intra_refresh=0 / rc_lookahead=60 / rc=2pass / mbtree=1 / bitrate=626 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / cplxblur=20.0 / qblur=0.5 / vbv_maxrate=20000 / vbv_bufsize=25000 / nal_hrd=none / ip_ratio=1.40 / aq=1:0.60
Encoded date                             : UTC 2018-04-08 12:00:12
Tagged date                              : UTC 2018-04-08 12:00:17
Codec configuration box                  : avcC

Audio
ID                                       : 2
Format                                   : AAC LC SBR
Format/Info                              : Advanced Audio Codec Low Complexity with Spectral Band Replication
Commercial name                          : HE-AAC
Format settings                          : Explicit
Codec ID                                 : mp4a-40-2
Duration                                 : 23 min 40 s
Bit rate mode                            : Variable
Bit rate                                 : 80.0 kb/s
Maximum bit rate                         : 99.4 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 23.438 FPS (2048 SPF)
Compression mode                         : Lossy
Stream size                              : 13.5 MiB (11%)
Encoded date                             : UTC 2018-04-08 11:35:59
Tagged date                              : UTC 2018-04-08 12:00:17

**Comment by Dendraspis** [comment]
1. This is an issue section, so use a forum for support.
2. What you need is ffmpeg, which should be better for your sort of problem - nevertheless StaxRip is also able to solve your problem.

**Comment by stax76** [comment]
> Is it possible NO re-encode video with staxrip (just use the direct video stream)?

Yes, in the same menu where you choose x265, there is a menu item Copy/Mux at the bottom.

**Comment by DJuego** [comment]
Thank you, **stax76**!! It was just what I needed. Forgive my foolishness. :-/ I close the "issue" happily.

And congrats on the new release of Staxrip! I've known Staxrip for quite a few years now, although I've used it very rarely. I love that it's absolutely portable. I have always believed that it makes the complicated world of multimedia encoding accessible. That it's free and open source is incredible. 

DJuego

P.S: Sorry for this question in your issue section. Is there a specially recommended place for support in this world? So far I have located the VideoHelp forum.
